cmd/main.go

```diff
@@ -30,15 +30,13 @@ import (
 )
 
 func main() {
```

Comment:
better approach is to use a hidden root command with sync as a subcommand. the package organization then properly reflects the commands as well

```suggestion
var rootCmd = &cobra.Command{
	Use:          "secret-sync",
	Short:        "secret-sync",
	Long:         `secret-sync`,
	SilenceUsage: true,
}

func init() {
	cobra.OnInitialize(
		initLogger,
	)
	
	rootCmd.AddCommand(sync.NewSyncCmd())
}

// Execute adds all child commands to the root command and sets flags appropriately.
func main() {
	// Create os.Signal aware context.Context which will trigger context cancellation
	// upon receiving any of the listed signals.
	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGHUP, syscall.SIGTERM)
	defer func() {
		cancel()
	}()

	cobra.CheckErr(rootCmd.ExecuteContext(ctx))
}
```

Reply:
I've created an issue that is linked to rewriting the command structure ( #202 ), I already started rewriting it, and it will be very similar to this, but I would like that to be part of another PR.